lib/Conv2Image.pm

```diff
@@ -19,6 +19,7 @@ require Exporter;
 
 use strict 'vars';
 use integer;
+use File::Temp;
 
 sub Conv2Image
```

Comment:
Like I said about CompressImage, but here it's more important:

Since we have to think about what exactly the `sub` does to get it tested, now is **the** time to write it down in the form of its API docs.
As a user of a library, I must not be forced to read the bodies of subs. Please write doc comments, ideally in a standard format understood by Perl docs tools.

Reply:
I plan to but one thing after another; this is still WIP.